pytorch_lightning/core/optimizer.py

```diff
@@ -199,7 +201,12 @@ def dis_closure():
 
             else:
                 with trainer.profiler.profile(profiler_name):
-                    optimizer.step(closure=closure, *args, **kwargs)
+                    print(optimizer)
```

Comment:
drop the print
```suggestion
```

Reply:
Great catch ! Miss this one when taking screenshot !